build_tools/github_actions/lint_markdown.sh

```diff
@@ -41,6 +43,6 @@ then
 fi
```

Comment:
`if ! command -v docker &> /dev/null`

Should we add something like this for shellcheck? Verify that it is installed at all? These lint scripts are intended to be run locally, so this would be useful

Reply:
Good idea. Done.